### PR TITLE
feat: add chaos mesh api description and table schema

### PR DESCRIPTION
**Pull request by kerenfi**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                      |
| Breaking change |✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |

Related issues: #634 

Further  information:

### CODE REVIEW

File: docs/openapi3.yaml [hunk 1]
```diff
@@ -1890,6 +1893,185 @@ paths:
...
+        - in: query
+          name: limit
+          description: Max results to return from this query ( Limit to 100 )
+          required: false
+          schema:
+            type: integer
+            maximum: 200
+            default: 100
+        - in: query
+          name: template
```
Comment (GuyAb) [inline docs/openapi3.yaml:1962]: Isn't it irrelevant? all are templates

File: docs/openapi3.yaml [hunk 2]
```diff
@@ -1890,6 +1893,185 @@ paths:
...
+          description: filter to get templates only
+          required: false
+          schema:
+            type: boolean
+            example: true
+        - in: query
+          name: exclude
```
Comment (GuyAb) [inline docs/openapi3.yaml:1969]: why do we need this?

File: docs/openapi3.yaml [hunk 3]
```diff
@@ -2956,3 +3138,70 @@ components:
...
+          type: string
+          minLength: 1
+          description: The name of the experiment.
+          example: Custom javascript for logging
+        kubeObject:
+          $ref: '#/components/schemas/kubeObject'
+        updated_at:
+          type: string
+          format: date-time
+          description: The date and time that the procesor file was updated.
```
Comment (GuyAb) [inline docs/openapi3.yaml:3162]: "procesor file" not relevant in this case

File: docs/openapi3.yaml [hunk 4]
```diff
@@ -2956,3 +3138,70 @@ components:
...
+        - spec
+      properties:
+        apiVersion:
+          type: string
+          enum:
+            - "chaos-mesh.org/v1alpha1"
+        kind:
+          type: string
+          enum:
+            - 'dnschaos'
```
Comment (GuyAb) [inline docs/openapi3.yaml:3184]: this one is small letters on purpose?

File: src/chaos-experiments/models/database/sequelize/sequelizeConnector.js [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    deleteChaosExperiment
+};
+
+async function init(sequelizeClient) {
+    client = sequelizeClient;
+    await initSchemas();
+}
+
+async function insertChaosExperiment(experimentId, experiment, contextId) {
+    const chaosExperimentModel = client.model('chaos-experiment');
```
Comment (GuyAb) [inline src/chaos-experiments/models/database/sequelize/sequelizeConnector.js:22]: shouldnt the function generate the id?

File: src/chaos-experiments/models/database/sequelize/sequelizeConnector.js [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    };
+
+    if (contextId) {
+        options.where.context_id = contextId;
+    }
+
+    return _getChaosExperiment(options);
+}
+
+async function deleteChaosExperiment(processorId) {
```
Comment (GuyAb) [inline src/chaos-experiments/models/database/sequelize/sequelizeConnector.js:83]: should be experiment id and not processor id

File: docs/openapi3.yaml (hunk 3 above)
Comment (kerenfi) [reply docs/openapi3.yaml:3162]: done

File: docs/openapi3.yaml (hunk 2 above)
Comment (kerenfi) [reply docs/openapi3.yaml:1969]: I'd just a common option in the crud sometimes not to show everything

File: docs/openapi3.yaml (hunk 4 above)
Comment (kerenfi) [reply docs/openapi3.yaml:3184]: fixed

File: src/chaos-experiments/models/database/sequelize/sequelizeConnector.js (hunk 5 above)
Comment (kerenfi) [reply src/chaos-experiments/models/database/sequelize/sequelizeConnector.js:22]: model generates it https://github.com/Zooz/predator/blob/chaos-experiments-api/src/chaos-experiments/models/chaosExperimentsManager.js#L18

File: src/chaos-experiments/models/database/sequelize/sequelizeConnector.js (hunk 6 above)
Comment (kerenfi) [reply src/chaos-experiments/models/database/sequelize/sequelizeConnector.js:83]: done

**Pull request closed without merging**